### ISSUE TITLE
adding values to header_pages results in no header pages shown

### ISSUE THREAD

**Issue by arielelkin** [jekyll/minima#253]
```bash
$ jekyll new test_minima_headers
$ cd test_minima_headers/
$ cp about.md about2.md  #create a new page
```

Now as per defaults, Jekyll will show **the two** About pages in the header. I only want to show `about.html` and not `about2.html` in the header. So I add the following lines to `_config.yml`:

```
header_pages:
  - about.html
```

I expect only one About page to show in the header, but the header is empty: 

![image](https://user-images.githubusercontent.com/1756909/41971114-3ef11b28-7a05-11e8-927b-0d81ec534543.png)

Should be pretty easy to reproduce. I've also uploaded a test repo here: https://github.com/arielelkin/test_minima_headers

Any thoughts?

**Comment by arielelkin** [comment]
@ashawley no, as you can see from the test repo. But even if I do the problem remains.

**Comment by ashawley** [comment]
Ah, I guess the `header_pages` is expecting a file path, and not the generated url.  The example in `_config.yml` is confusing.  I've tried to fix it in #254.

**Comment by arielelkin** [comment]
Ah! So it's expecting the file path _in the source_, not the rendered HTML. Thanks!
